docs/_guide/06-template-reference.md

```diff
@@ -420,6 +424,38 @@ const template = html`
 
 In this case, the `immutableItems` array is mapped over only when the array reference changes.
 
+### live
+
+`attr=${live(value)}`
+
+Location: attribute or property bindings
+
+Checks binding value against the _live_ DOM value, instead of the previously
+bound value, when determining whether to update the value.
+
+This is useful for cases where the DOM value may change from outside of
+lit-html. For example, when binding to an `<input>` element's `value` property,
+a content editable element's text, or to a custom element that changes its
+own properties or attributes.
+
+In these cases if the DOM value changes, but the value set through lit-html
+bindings hasn't, lit-html won't know to update the DOM value and will leave
+it alone. If this is not what you want—if you want to overwrite the DOM
```

Comment:
Add spaces here: "want-if", and should it be balanced with another "-" at the end of the clause?

Reply:
Our house style is to use em dashes with no spaces. These do look weird in monospace fonts because the em dash is the same size as a hyphen, but it looks good when it's set in a normal font.

https://developers.google.com/style/dashes
Example in the current docs: https://lit-html.polymer-project.org/guide/template-reference#template-structure

Reply:
Added the closing dash, updated the API docs to match.